.github/workflows/worker.yml

```diff
@@ -27,12 +34,26 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
this configures to download the full history of *all* branches ; checkout time jumped a bit -
what if its being set to some value like 500 or 1000 ? would that make the jacoco work correctly?

Reply:
Tried setting it to 500, seems to work.

Reply:
checkout time is back to 8s! :)